build.mjs

```diff
@@ -86,12 +86,16 @@ function isImageLgxCompatible(image) {
             return false;
         }
     } else {
-        return true;
+        if (image.hasOwnProperty('path')) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
We could do without the branch here, right?
```suggestion
        return image.hasOwnProperty('path'));
```

Reply:
I guess.